Cannot find module '../types'` in @ngx-rock/yandex-smart-captcha (both 18.0.0 and 19.1.0)

Hello,

After installing `@ngx-rock/yandex-smart-captcha` (tried both `18.0.0` and latest `19.1.0`) in my Angular 18 project, I'm encountering this error on build:

```
Error: node_modules/@ngx-rock/yandex-smart-captcha/lib/yandex-smart-captcha/yandex-smart-captcha.component.d.ts:3:62 - error TS2307: Cannot find module '../types' or its corresponding type declarations.

3 import { JavascriptErrorPayload, SmartCaptchaInstance } from '../types';
                                                               ~~~~~~~~~~
```

My project dependencies (relevant parts):
```json
"dependencies": {
  "@angular/core": "18.1.3",
  "@angular/common": "18.1.3",
  "@angular/forms": "18.1.3",
  "@ngx-rock/yandex-smart-captcha": "^18.0.0",
  ...
}
```
Tried also with `"@ngx-rock/yandex-smart-captcha": "^19.1.0"`, same error.

It seems that the `../types` module is either missing from the published NPM package or incorrectly referenced.

---

**Steps to reproduce:**
1. Install `@ngx-rock/yandex-smart-captcha` (`18.0.0` or `19.1.0`)
2. Run `ng serve`
3. Build fails with the error above.

---

**Expected behavior:**  
The package should compile without missing module errors.

**Current workaround:**  
None yet.

Could you please fix the missing `types` module or adjust the import paths?

Thanks!

--- a/libs/yandex-smart-captcha/src/lib/types/index.ts
+++ b/libs/yandex-smart-captcha/src/lib/types/index.ts
@@ -1,4 +1,6 @@
 // Declare global window interface
+export {};
+
 declare global {
   interface Window {
     smartCaptcha: SmartCaptchaInstance;

